develop/tutorials/articles/220-internationalization/01-localizing-your-application.markdown

```diff
@@ -197,10 +206,12 @@ bundle.
 
         Provide-Capability: liferay.resource.bundle;resource.bundle.aggregate:String="(bundle.symbolic.name=com.liferay.my.editor.lang)";resource.bundle.base.name="content.Language"
 
-It might seem a bit strange that the resource bundle aggregate has only one
-bundle. Grouping together multiple resource bundles comes into play when you
-want to use your module's own resource bundle *in addition to* a language module
-resource bundle. 
+In this case, the web module solely uses the language module's resource bundle. 
+
+## Using Other Resource Bundles in Addition to Your Own [](id=using-other-resource-bundles-in-addition-to-your-own)
```

Comment:
It's important to use "Your Own", or something to that effect, as your own module's resource bundle is used and prioritized above those of the other resource bundles.

The previous section does NOT involve using your own module's resource bundle.